streamlit_echarts/frontend/src/EchartsChart.tsx

```diff
@@ -54,6 +54,7 @@ const EchartsChart = (props: ComponentProps) => {
       let match = funcReg.exec(v)
       if (match) {
         const funcStr = match[1]
+        // eslint-disable-next-line
```

Comment:
this is when I run npm start, the terminal ouput this, tell me to add this line. but When I run this at home, it disappear...I guess it's because VS environment I set up.